app/assets/stylesheets/custom.css.scss

```diff
@@ -36,7 +42,7 @@ body {
 }
 
 .progress {
-  background-color: white !important;
+  background-color: $primaryColor !important;
```

Comment:
!important should not be used<br>Name of variable `primaryColor` should be written in all lowercase letters with hyphens instead of underscores